### PR TITLE
Refactor : Event List Item UI

### PR DESCRIPTION
**Pull request by kumarpalsinh25**
Fixes, #1395 

Event List - Skeleton UI:
Frame 1 (1)

References:
All Events
Dashboard Events
Space Evens Tab
Space Overview

### CODE REVIEW

**Comment by gnunicorn** [comment]
oh, so pretty!

File: app/lib/features/events/widgets/events_list.dart [hunk 1]
```diff
@@ -21,43 +21,35 @@ class EventsList extends ConsumerWidget {
...
+                      onPressed: () =>
+                          context.pushNamed(Routes.calendarEvents.name),
+                      child: Text('See all my ${events.length} events'),
+                    ),
+                  )
+                : const SizedBox.shrink(),
+          ],
         );
       },
       loading: () => const CircularProgressIndicator(),
```
Comment (gnunicorn) [inline app/lib/features/events/widgets/events_list.dart:55]: using a skeleton with a few items similar to the new design would have been nice(r) here ...

**Review by gnunicorn — APPROVED** [review]

File: app/lib/features/events/widgets/events_list.dart (hunk 1 above)
Comment (kumarpalsinh25) [reply app/lib/features/events/widgets/events_list.dart]: Added skeleton UI :)
Frame 1 (1)

Comment (gnunicorn) [reply app/lib/features/events/widgets/events_list.dart]: oh, so pretty :glowing_eyes: .

**Pull request merged**